### PR TITLE
Activating feeds and articles

### PR DESCRIPTION
**Pull request by juri-sinitson**
Hi,

with this feature the feeds and articles you clicked are marked. Thus you can better see which feed and article you are currently on. All Feeds, Favourites, Unread Articles, Recently Read and Saved Articles are also marked when clicked.

Best regards
Juri

### CODE REVIEW

**Comment by mrgodhani** [comment]
Great work thanks for contribution. Only thing is have to also find a color for Dark mode. And yeah obviously resolving conflict which I can help.

**Comment by juri-sinitson** [comment]
Hi Meet,

thank you for your feedback. I'll accomplish the pending tasks of this branch the next week.

Best Regards
Juri

**Comment by juri-sinitson** [comment]
Hi Meet,

I have adjusted the active color for color mode. I have also made the default and dark mode color adjustment more DRY using CSS-variables.

I had to revert two previous commits due to errors.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,18 @@
 {
   "name": "raven-reader",
-  "version": "0.5.1",
+  "version": "0.4.5",
```
Comment (juri-sinitson) [inline package.json]: Which Version would be reasonable when I reverted two commits?

Comment (mrgodhani) [reply package.json]: 0.5.1

**Comment by juri-sinitson** [comment]
@mrgodhani 
I have corrected the version to "0.5.1".
Is it ok for you to take this pull request to master with older versions of electron and eslint-plugin-html and then to solve the version issues later?

**Comment by mrgodhani** [comment]
> @mrgodhani
> I have corrected the version to "0.5.1".
> Is it ok for you to take this pull request to master with older versions of electron and eslint-plugin-html and then to solve the version issues later?

Ye thanks for contributing I will merge and update

**Pull request merged**